src/libdmr/compositing_manager.cpp

```diff
@@ -64,8 +64,7 @@ class PlatformChecker
                     qInfo() << "match shenwei";
                     _pf = Platform::Alpha;
 
-                } else if (machine.find("mips") != string::npos
-                           || machine.find("loongarch64") != string::npos) { // loongson
+                } else if (machine.find("mips") != string::npos) { // MIPS-based Loongson
```

Comment:
This is the playback flow following MIPS, but the current architecture requires rendering in a non-OpenGL manner.